docs/conf/portland/2018/news/volunteers-shirts-tickets.rst

```diff
@@ -0,0 +1,66 @@
+:template: {{year}}/{{templatecode}}/generic.html
+
+.. post:: March 19, 2018
+   :tags: portland-2018, volunteers, shirts, tickets
+
+Call for Volunteers, Shirts on Sale, Tickets Update
+===================================================
+
+Greetings, fellow documentarians! The Portland conference is just short of two months away, and we can't wait to see you all soon! Here are the latest updates from the conference team to help you prepare.
+
+Shirts on sale
+--------------
+
+As you probably noticed, we are not including shirts with your conference ticket this year.
+This is due partially to the logistics of sizing, ordering, and managing stock, but also because of the inevitable leftovers that we find ourselves with every year.
+
+Inspired by our friends at DjangoCon US, we decided to shake things up and launch a separate sales channel for shirts, if you have your heart set out on them.
```

Comment:
`sales channel` is weird business speak. "we decided to shake things up and launch sell shirts separately" or similar.

Reply:
"set out on them" is also weird grammar.